src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 4.4.0
 -----
 
+ * fixed automatic adding of newlines when writing to output sections
```

Comment:
For bugfixes, the changelog entries are auto generated at release time. As such this change should be reverted